src/rtsp.cpp

```diff
@@ -644,6 +639,10 @@ namespace rtsp_stream {
     server.session_clear(launch_session_id);
   }
 
+  /**
+   * @brief Get the number of active sessions.
+   * @return Count of active sessions.
+   */
```

Comment:
The documentation blocks only need to be in the header assuming the object/function is defined there.